src/components/header/header.js

```diff
@@ -0,0 +1,57 @@
+import '../../vendor/polyfills/Function/prototype/bind'
+import '../../vendor/polyfills/Event' // addEventListener and event.target normaliziation
+
+function Header ($module) {
+  this.$module = $module
+}
+
+Header.prototype.init = function () {
+  // Check for module
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+
+  // Check for button
+  var $toggleButton = $module.querySelector('.js-header-toggle')
+  if (!$toggleButton) {
+    return
+  }
+
+  // Handle $toggleButton click events
+  $toggleButton.addEventListener('click', this.handleClick.bind(this))
+}
+
+/**
+* Toggle class
+* @param {object} node element
+* @param {string} className to toggle
+*/
+Header.prototype.toggleClass = function (node, className) {
+  if (node.classList.contains(className)) {
+    node.classList.remove(className)
```

Comment:
I think we need a polyfill for IE9 support.

I don't think too many people would notice since IE9 users typically is used on desktop machines but better to be safe.